app/lib/getNearbyServices.js

```diff
@@ -1,27 +1,49 @@
 const log = require('../../app/lib/logger');
-const docDB = require('../../config/config').docDB;
-const queries = require('../lib/queries');
-const filterServices = require('../lib/filterServices');
+const MongoClient = require('mongodb').MongoClient;
+const mongodbConfig = require('../../config/config').mongodb;
+const filterServices = require('./filterServices');
+const VError = require('verror').VError;
 
-function getNearbyServices(req, searchPoint, limits, next) {
-  const geoSpatialQuery = queries.geoSpatialQuery(searchPoint, limits.searchRadius);
+const connectionString = mongodbConfig.connectionString;
 
-  req.app.locals.docDBClient
-    .queryDocuments(docDB.collectionUrl, geoSpatialQuery)
-    .toArray((err, results) => {
-      if (err) {
-        log.error({ err }, 'DocumentDB query failed');
-        next(err);
-      } else {
-        log.debug(`${results.length} RESULTS returned...`);
+function getNearbyServices(searchPoint, limits, next) {
+  MongoClient.connect(connectionString).then((db) => {
+    const col = db.collection(mongodbConfig.collection);
 
-        log.debug('filter-and-sort-services-start');
-        const filteredServices = filterServices(results, limits);
-        log.debug('filter-and-sort-services-end');
+    col.aggregate([{
+      $geoNear: {
+        near: { type: 'Point', coordinates: searchPoint.coordinates },
+        distanceField: 'dist',
+        maxDistance: 32180,
+        distanceMultiplier: 0.001, // this is in miles
```

Comment:
Good one @markysoft, thanks for spotting. This answer has a better explanation than the docs: http://stackoverflow.com/questions/33864461/mongodb-print-distance-between-two-points